docs/nginx.rst

```diff
@@ -124,3 +124,93 @@ Create a uWSGI configuration which references the Sentry configuration::
 Finally, re-configure supervisor to run uwsgi instead of 'sentry start'::
 
   /www/sentry/bin/uwsgi --ini /www/sentry/uwsgi.ini
+
+
+Hosting Sentry at a Subpath
+----------------------------
+
+If your web server is hosting several applications then hosting Sentry at '/' may not be feasible for you. It is possible to configure your webserver such that all traffic going to '/sentry' can be directed at Sentry and everything else can remain as is.
+
+
+Subpath with uWSGI
+^^^^^^^^^^^^^^^^^^
+
+The easiest configuration is if you plan on configuring Nginx to proxy requests to uWSGI because uWSGI supports this with a simple configuration option. (Source: `uWSGI - Hosting multiple apps <http://uwsgi-docs.readthedocs.org/en/latest/Nginx.html#hosting-multiple-apps-in-the-same-process-aka-managing-script-name-and-path-info>`_)
+
+**uWSGI Configuration**
+
+If you are using a uWSGI configuration file, add these lines::
+
+    ; Host Sentry at /sentry
+    mount = /sentry=path/to/sentry/wsgi.py
+    manage-script-name = true
+
+If you call uWSGI directly, possibly from Supervisor, see :ref:`performance-web-server`.
+
+Edit the command used to start sentry by removing this option::
+
+    --wsgi-file getsentry/wsgi.py
+
+And adding these options::
+
+    --mount "/sentry=getsentry/wsgi.py" --manage-script-name
+
+**Nginx Configuration**
+
+Make the adjustment to the 'location' directive in Nginx's configuration file::
+
+    location '/' => location '/sentry'
+
+
+Subpath with Sentry's Default Webserver
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This is a bit more involved and harder to debug because not only does Nginx have to know the new location to listen for, it also must do rewriting of the urls it sends to the Sentry's built in server because it knows nothing about you trying to host the application elsewhere.
+
+These are not full configurations, but abbreviated versions to highlight the differences from the main examples.
+
+**Nginx Configuration**::
+
+    server {
+
+      ...
+
+      if ($http_referer ~ .com/sentry) {
+        rewrite /_static/(.*)$ /sentry/_static/$1 break;
+        rewrite /sentry/sentry/(.*)$ /sentry/$1 break;
+      }
+
+      location /sentry {
+        proxy_pass http://0.0.0.0:9000;
+        proxy_redirect off;
+
+        proxy_set_header       Host    $host;
+        proxy_set_header       SCRIPT_NAME     /sentry;
+        proxy_set_header       X-Real-IP       $remote_addr;
+        proxy_set_header       X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header       X-Forwarded-Proto       $scheme;
+      }
+
+      ...
+    }
+
+**Django Configuration**
+
+Your Sentry settings file ($SENTRY_CONF python file)::
+
+    ...
+
+    ################
+    ## Web Server ##
+    ################
+
+    # You MUST configure the absolute URI root for Sentry:
+    SENTRY_URL_PREFIX = 'http://blah.com/sentry'  # No trailing slash!
```

Comment:
if we know this is always how things are configured we could set FORCE_SCRIPT_NAME automatically in the setup stuff (in sentry/utils/runner.py)